TUTORIAL.md

```diff
@@ -147,6 +147,16 @@ type Value struct {
 }
 ```
 
+Since we want to parse also integers and the default lexer makes a difference,
```

Comment:
Since this is a tutorial, instead of replacing directly line 146 I added this paragraph.
If you prefer brevity, we can just remove it.